docs/modules/kaolin.ops.spc.rst

```diff
@@ -0,0 +1,151 @@
+.. _kaolin.ops.spc:
+
+kaolin.ops.spc
+==============
+
+.. _spc:
+
+Structured Point Clouds
+-----------------------
+
+Structured Point Clouds are a tensor representation of a point cloud hierarchy 
+:math:`P` defined by a sparse octree tensor :math:`O`.
```

Comment:
Is there a direct relation between `O` / `P` and the real variables that we are using? Shouldn't we use those real variables names instead?

I find the transition between the two notations a little bit abrupt.